code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -96,9 +97,21 @@
 	var/death_sound
 	var/death_message = "falls limp and stops moving..."
 	var/death_message_range = 2
+	var/list/scream_emote = list("screams!")
 	var/knockout_message = "has been knocked unconscious!"
 	var/halloss_message = "slumps to the ground, too weak to continue fighting."
 	var/halloss_message_self = "You're in too much pain to keep going..."
+	var/list/pain_messages = list("It hurts so much", "You really need some painkillers", "Dear god, the pain") // passive message displayed to user when injured
+	var/list/pain_item_drop_cry = list("screams in pain and ", "lets out a sharp cry and ", "cries out and ")
+
+	// External Organ Pain Damage
+	var/organ_low_pain = "<b>Your PARTNAME hurts.</b>"
+	var/organ_med_pain = "<b><font size=3>Your PARTNAME hurts badly!</font></b>"
+	var/organ_high_pain = "<b><font size=3>Your PARTNAME is screaming out in pain!</font></b>"
+
+	var/organ_low_burn = "<span class='danger'>Your PARTNAME burns.</span>"
+	var/organ_med_burn = "<span class='danger'><font size=3>Your PARTNAME burns horribly!</font></span>"
+	var/organ_high_burn = "<span class='danger'><font size=4>Your PARTNAME feels like it's on fire!</font></span>"
```

Comment:
These var names should be changes so its clear from the name that this is a message and not some sort of value.